deepmd/pt/utils/stat.py

```diff
@@ -200,3 +238,153 @@ def model_forward_auto_batch_size(*args, **kwargs):
             stat_file_path.save_numpy(bias_atom_e)
     assert all(x is not None for x in [bias_atom_e])
     return to_torch_tensor(bias_atom_e)
+
+
+def compute_output_stats_with_atomic(
```

Comment:
I find the current implementation has too much hard coding, which makes it not easily extensible to new fitting targets.